docs/guides/ssr.md

```diff
@@ -18,10 +18,19 @@ The exact implementation of these mechanisms may vary from platform to platform,
 - Static Generation (SSG)
 - Server-side Rendering (SSR)
 
+> Only Server-side Rendering (SSR) is presently supported by React Server Components in Next.js 13's experimental `app` directory.
+
 React Query supports both of these forms of pre-rendering regardless of what platform you may be using
 
 ### Using `initialData`
 
+The setup is minimal and this can be a quick solution for some cases, but there are a **few tradeoffs to consider** when compared to the full approach:
```

Comment:
Love this section! It's been sorely missing, glad you included it, and well written. :smile: